### PR TITLE
fix: [CDS-68671]: modified placeholders for loading

### PR DESCRIPTION
**Pull request by sahithikolichala**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

#### [Contributor license agreement](https://github.com/harness/harness-core-ui/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

https://user-images.githubusercontent.com/57372986/236551402-1223e816-66ef-4bd0-b52f-d33d1c2fbefc.mov

### CODE REVIEW

**Comment by sahithikolichala** [comment]
retrigger cypress-cd

**Comment by sahithikolichala** [comment]
retrigger titlecheck

**Comment by sahithikolichala** [comment]
retrigger cypress-cd

File: src/modules/70-pipeline/components/ArtifactsSelection/ArtifactRepository/ArtifactLastSteps/Nexus3Artifact/Nexus3Artifact.tsx [hunk 1]
```diff
@@ -104,14 +104,34 @@ export function Nexus3Artifact({
 }: StepProps<ConnectorConfigDTO> & ImagePathProps<Nexus2InitialValuesType>): React.ReactElement {
   const { getString } = useStrings()
   const isIdentifierAllowed = context === ModalViewFor.SIDECAR || !!isMultiArtifactSource
-  const { CDS_NEXUS_GROUPID_ARTIFACTID_DROPDOWN } = useFeatureFlags()
+  // const { CDS_NEXUS_GROUPID_ARTIFACTID_DROPDOWN  = true} = useFeatureFlags()
```
Comment (mtran7) [inline src/modules/70-pipeline/components/ArtifactsSelection/ArtifactRepository/ArtifactLastSteps/Nexus3Artifact/Nexus3Artifact.tsx:107]: can remove?

**Comment by mtran7** [comment]
Should allow for custom values if the group id and other fields are taking a while to fetch. User may know the value and continue without being blocked by the call

**Comment by sahithikolichala** [comment]
retrigger jest

**Review by mtran7 — APPROVED** [review]

**Comment by sahithikolichala** [comment]
retrigger jest

File: src/modules/70-pipeline/components/ArtifactsSelection/ArtifactRepository/ArtifactLastSteps/Nexus3Artifact/__tests__/__snapshots__/Nexus3Artifact.test.tsx.snap [hunk 2]
```diff
@@ -665,4886 +665,3 @@ exports[`Nexus Artifact tests form renders correctly in Edit Case 1`] = `
   </div>
 </div>
 `;
-
-exports[`Nexus Artifact tests render form correctly when repositoryFormat is Maven 1`] = `
```
Comment (pankajsingh-harness) [inline src/modules/70-pipeline/components/ArtifactsSelection/ArtifactRepository/ArtifactLastSteps/Nexus3Artifact/__tests__/__snapshots__/Nexus3Artifact.test.tsx.snap:669]: Please check if these snaps are removed intentionally or by mistake?

**Pull request closed without merging**